### ISSUE TITLE
Finetuning scheme

### ISSUE THREAD

**Issue by shubhamjha97** [HsinYingLee/OPN#9]
How do you finetune on the action recognition datasets? The datasets contain videos. Do you randomly extract frames from the videos? What is the size of the frames that you use?

**Comment by HsinYingLee** [comment]
Yes, I randomly select one frame each time. The frame size is 224. The initial learning rate is 0.01, I drop the learning rate by 0.1 when the validation loss does not decrease anymore.
